src/Translator.php

```diff
@@ -63,7 +64,7 @@ public function exists($key)
      */
     public function plural($key, $count, array $vars = [])
     {
-        if ($count == 1) {
+        if ($count === 0 || $count === 1) {
```

Comment:
Updated phpdoc.
I think that properties of `ChronosInterval` are always integer. correct?